### PR TITLE
Include 'oc set deployment-hook' and 'oc set build-hook' in doc

### PR DESCRIPTION
**Pull request by csrwng**
Adds CLI instructions for deployment and build hooks.

### CODE REVIEW

**Comment by csrwng** [comment]
@bparees ptal

**Comment by bparees** [comment]
lgtm, @openshift/team-documentation ptal

File: dev_guide/builds.adoc [hunk 1]
```diff
@@ -1734,6 +1734,26 @@ Providing both `*script*` and `*command*` simultaneously creates an invalid
 build hook.
 ====
 
+==== Using the command line
```
Comment (ahardin-rh) [inline dev_guide/builds.adoc]: Title case for the full heading

File: dev_guide/builds.adoc [hunk 2]
```diff
@@ -1734,6 +1734,26 @@ Providing both `*script*` and `*command*` simultaneously creates an invalid
...
+
+The `*oc set build-hook*` command can be used to set the build hook for a build config.
```
Comment (ahardin-rh) [inline dev_guide/builds.adoc]: s/ `*oc set build-hook*`/`oc set build-hook`
s/config/configuration

File: dev_guide/deployments.adoc [hunk 3]
```diff
@@ -552,6 +552,19 @@ pod will have the following properties:
 * The hook failure policy is `Abort`, meaning the deployment will fail if the hook fails.
 * The hook pod will inherit the `data` volume from the deployment configuration pod.
 
+==== Using the command line
```
Comment (ahardin-rh) [inline dev_guide/deployments.adoc]: Title case for the full heading

File: dev_guide/deployments.adoc [hunk 4]
```diff
@@ -552,6 +552,19 @@ pod will have the following properties:
...
+
+The `*oc set deployment-hook*` command can be used to set the deployment hook for a
+deployment config. For the example above, you can set the pre-deployment hook with the
```
Comment (ahardin-rh) [inline dev_guide/deployments.adoc]: No asterisks around oc set build-hook, just back ticks
s/config/configuration

File: dev_guide/deployments.adoc (hunk 3 above)
Comment (ahardin-rh) [inline dev_guide/deployments.adoc]: Please add the following anchor above the title:
[[deployments-using-the-command-line]]

File: dev_guide/builds.adoc (hunk 1 above)
Comment (ahardin-rh) [inline dev_guide/builds.adoc]: Please add the following anchor above the title:
[[builds-using-the-command-line]]

**Comment by ahardin-rh** [comment]
@csrwng @bparees just a few comments in line. Please confirm that my labels and milestone are correct. Thanks!

**Comment by csrwng** [comment]
@ahardin-rh thx for the quick review. I've addressed your comments.

For the labels, this is meant to go in the next release of OpenShift, so post-3.2 and it does apply to all of them (dedicated, enterprise, and online).

**Comment by ahardin-rh** [comment]
@csrwng Thanks! Looks good! Also, thanks for your guidance on the labels.

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../dev_guide/builds.html[Builds]
|Added information about the link:../dev_guide/builds.html#build-hooks-using-the-command-line[`oc set build-hook` command].
%
|link:../dev_guide/deployments.html[Deployments]
|Added information about the link:../dev_guide/deployments.html#deployment-hooks-using-the-command-line[`oc set deployment-hook` command].
%

**Pull request merged**